packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -357,23 +358,23 @@ export function FileUploader({
       >
         {fileStatuses?.map((status, index) => (
           <Tracker
-            percentage={status.percentage}
+            errorMessage={status?.fileErrors}
             file={status.file}
-            hasImage={status.file?.type.startsWith('image/')}
-            url={URL.createObjectURL(status.file)}
+            fileState={status?.fileState}
+            hasImage={status.file?.type.startsWith('image/') && showImages}
```

Comment:
This the change I added. So now it has to have an image and showImages has to be true.

Reply:
I think we need to change this logic a bit. I was thinking this prop would show or hide both the image and file icon depending if it is true or false. If `showImages` is true, there will always be either an image or icon. If `showImages` is false, no image or icon will appear. Does that make sense?

Reply:
OK! @dbanksdesign I can make it so it doesn't show the icon either!

Reply:
Fixed in [cf8a443](https://github.com/aws-amplify/amplify-ui/pull/2988/commits/cf8a443a1d574255cda35fa8fdb5f8a3b4585418)